### ISSUE TITLE
Code Rubric

### ISSUE THREAD

**Issue by Card0154** [Web-Dev-6-Mega/mega-app#14]
| | 0 points | 1 points | 2 points | 3 points |
| --- | --- | --- | --- | --- |
| **Accessibility** | Not accessible | Basic attributions in some content | Basic attributions in all content | Exemplary attribution in all content |
| **Code quality** | Incomplete | Lots of semantic and validation errors | Minimal errors and barely SEO compatible | No errors and SEO compatible |
| **Work Distribution** | One person contributed everything | One person did not contribute anything | Everyone made minimal contributions | Everyone actively participated and contributed |
| **Communication** | No communication | Minimal communication | Decent communication | Constant, effective communication |
| **Responsiveness** | Not responsive | Responsiveness is clunky | Responsive with minimal errors | Fully responsive and clean design |

**Comment by Sabrinaheideman** [comment]
Accessibility: 1 Point. Not all content was attributed properly.

Code Quality: 2 Points. Not completely SEO compatible. 

Work Distribution: 3 Points. Everyone participated equally.

Communication: 3 Points. Communication was very good.

Responsive: 3 Points. Fully responsive and clean.
